.vscode/launch.json

```diff
@@ -37,7 +49,7 @@
       "args": [
         "generate"
       ],
-      "cwd": "${workspaceRoot}/examples/generator-example",
+      "cwd": "${workspaceRoot}/examples/generator-full-stack-example",
```

Comment:
`examples/generator-full-stack-example` could be `examples/generator-full-stack` if you change the name. No need (IMO) to repeat "example".

Reply:
Perfect!

Reply:
Feel free to rename runtime-example as well in your PR :)